packages/mui-base/src/Switch/index.ts

```diff
@@ -1,5 +1,10 @@
 'use client';
-export { Switch } from './Switch';
+import { combineComponentExports } from '../utils/combineComponentExports';
+import { Switch as SwitchRoot } from './Switch';
+import { SwitchThumb } from './SwitchThumb';
+
 export * from './Switch.types';
 
-export * from './switchClasses';
+export const Switch = combineComponentExports(SwitchRoot, {
+  Thumb: SwitchThumb,
+});
```

Comment:
The API docs show `import { SwitchThumb } from '@mui/base/SwitchThumb'` being possible. Should this be the case, or just have the generator updated to show `Switch.Thumb` usage?

Reply:
It's an issue with the API docs generator I haven't fixed yet, but we can do this in a separate PR.